views/index.pug

```diff
@@ -2,8 +2,35 @@ extends layout
 
 block content
 
-  h1=title
+  div.container
 
-  #root
-  script(src="/public/javascripts/index.js")
+    div.spotlight-container
+     div.spotlight-container-img
+      div.spotlight-header LIVE MONITOR DASHBOARD SOFTWARE
+       div.spotlight-message Focus Your Team and Improve Your KPIs
+        div.buttons
+          button.btn.btn-default(type='button') Start Your Free Trial
+          button.btn-demo(type='button') Book A Demo
+      div.spotlight-container-text
 
+    div.content
+     h3 Easy APIs to build dashboards for your next project
+     h4 Quickly see the information important to your team
+
+    div.content-tools
+      div.feature
+        div.feat-1
+          a(href='/product') This is Feature 1
+          p Real Time Data Visualiztion
+
+      div.feature
+        div.feat-2
+          a(href='/product') This is Feature 2
+          p Real Time Project Updates
+
+      div.feature
+        div.feat-3
+          a(href='/product') This is Feature 3
+          p Real Time Sales Goals and Effective Time Management
+
+      script(src="/public/javascripts/index.js")
```

Comment:
If this is the global js file, it should be included in the layout, last in body... would prefer not to see script includes in individual view files, as this means they might be included in a tag, rather than at the end of the body